.github/workflows/ci_tests.yml

```diff
@@ -0,0 +1,29 @@
+name: CI tests
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Read .nvmrc
+        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+        id: nvm
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "${{ steps.nvm.outputs.NVMRC }}"
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Check types
+        run: yarn tsc
```

Comment:
This should have `--noEmit` on it so that we don't write any files, since I didn't set up the tsconfig with that in mind, so it might mess something else up (like jest).

Reply:
Fixed